content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -218,29 +316,35 @@ kubectl autoscale rs frontend --max=10
 
 ## Alternatives to ReplicaSet
 
-### Deployment (Recommended)
+### Deployment (recommended)
 
-[`Deployment`](/docs/concepts/workloads/controllers/deployment/) is a higher-level API object that updates its underlying ReplicaSets and their Pods
-in a similar fashion as `kubectl rolling-update`. Deployments are recommended if you want this rolling update functionality,
-because unlike `kubectl rolling-update`, they are declarative, server-side, and have additional features. For more information on running a stateless
-application using a Deployment, please read [Run a Stateless Application Using a Deployment](/docs/tasks/run-application/run-stateless-application-deployment/).
+[`Deployment`](/docs/concepts/workloads/controllers/deployment/) is an object which can own ReplicaSets and update
```

Comment:
BTW, @dtornow found out that Deployments do ensure that the Pods in the resulting ReplicaSets match the image spec listed in the Deployment manifest.